pkg/crd/kubermatic/v1/datacenter.go

```diff
@@ -252,7 +252,7 @@ type DatacenterSpec struct {
 
 	//nolint:staticcheck
 	//lint:ignore SA5008 omitgenyaml is used by the example-yaml-generator
-	Fake *DatacenterSpecFake `json:"fake,omitempty,omitgenyaml"`
+	Fake *DatacenterSpecFake `json:"fake,omitempty,omitempty"`
```

Comment:
Ok, since you've removed omitgenyaml, you also should remove the comment about this tag, isn't it?

Reply:
wow i have no idea where this comes from :o

Reply:
I reverted the crd change